### DISCUSSION TITLE
nRF5340 DK / permit_with_key?

### DISCUSSION THREAD

**Discussion by JohnConnett** [kardia-as/zigpy-zboss#43] [Q&A]
I have an nRF5340 DK. Has anyone successfully used **zigpy-zboss** with one of these? I presume I would have to compile the ZBOSS NCP Host firmware image myself configured to use maximum memory?

From a recent change to `zigpy_zboss/zigbee/application.py` it appears that adding a device using an install code is not implemented. Is that a limitation of the ZBOSS NCP Host firmware?

**Comment by Hedda** [comment]
@JohnConnett  If you can isolate each separate question then suggest that it is probably better if you ask such under issues instead -> https://github.com/kardia-as/zigpy-zboss/issues (as is posted separately for tracking then you can clarify all quantify questions further)

Update #1: Posted the later question about install code support as a separate issue here:

- https://github.com/kardia-as/zigpy-zboss/issues/45

Update #2: Posted the first question about nRF5340 support as a separate issue here:

- https://github.com/kardia-as/zigpy-zboss/issues/46
